frontend/src/constants/Styling.js

```diff
@@ -1,9 +1,29 @@
 const MODAL_MAX_HEIGHT = '800px';
 const PAGE_HEIGHT = '1140px';
-const TOPICS_TREE_HEIGHT = 500;
-const TOPICS_TREE_WIDTH = 1000;
+
+const TOPICS_TREE_HEIGHT = 650;
```

Comment:
If we have problems with height being to small, we can increase this a lot. I'd expect a big tree to take even the whole page height (although having it as small as possible is a good thing)